kata-deploy/scripts/kata-deploy.sh

```diff
@@ -45,6 +51,20 @@ function install_artifacts() {
 	chmod +x /opt/kata/bin/*
 }
 
+function restart_cri_runtime() {
+	if [ "$1" == "containerd-k3s" ]; then
+		if systemctl is-active --quiet k3s-agent; then
+			# worker node
+			systemctl restart k3s-agent
+		else
+			# master node
+			systemctl restart k3s
+		fi
```

Comment:
debating whether just restarting both k3s and k3s-agent will make life easier. Either way, we can't just do restart $1, so perhaps we shouldn't care...

Reply:
or we could rename `containerd-k3s` to `k3s || systemctl restart k3s-agent` :grimacing:
Assuming that's a terrible idea, I like your suggestion to have `get_container_runtime` return `k3s` or `k3s-agent`